Makefile

```diff
@@ -9,6 +9,8 @@ KUBERNETES_VERSION ?= 1.23.0
 ENVTEST_K8S_VERSION ?= 1.23
 GOLANGCI_LINT_VERSION := 1.43.0
 
+PLATFORM ?= linux/amd64
```

Comment:
One last nit, default value for this can be `linux`.
This way if you are building on arm64 it doesn't try to build amd64.

Reply:
Oh thanks, that's great to know!